webdriver-spec.html

```diff
@@ -5200,7 +5217,7 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+ <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
```

Comment:
Hm, yes. Because a frame’s document is separate from that of the owner document if in an `<iframe>`.